src/main.cpp

```diff
@@ -344,7 +344,7 @@ esp_err_t _http_event_handler(esp_http_client_event_t *evt)
         Log->println("HTTP_EVENT_ON_HEADER");
         break;
     case HTTP_EVENT_ON_DATA:
-        Log->println("HTTP_EVENT_ON_DATA");
+        Log->print(".");
```

Comment:
Confirm if this syntax is correct or needs to be updated?

```suggestion
        Log->print(F("."));
```

Reply:
Hi. The F() macro is kept only for backwards compatibility on the ESP32 and can (or better should) be dropped. It's still in many places in the code, I usually remove it when I touch the code.